### ISSUE TITLE
[Feature]: 7001 Support

### ISSUE THREAD

**Issue by 732502** [hack-different/ipwndfu#71]
### Brief Description

When you create a GitHub issue, the SoC/Chip ID doesn't have 7001 but the next branch supports this.

### Problem Solved

When you create a GitHub issue, the SoC/Chip ID doesn't have 7001 but the next branch supports this.

### Proposed Design

7001 added to the soc/chip id

### Testing Plan

No testing, just add it.

### I'm building this

- [X] I volenteer to own this feature

**Comment by TheRealClarity** [comment]
T7001 Support is already added on the Next branch. Closing.

**Comment by DesentivsedToGore** [comment]
> T7001 Support is already added on the Next branch. Closing.

no, i mean if you create a github issue in the chip section there isnt a 7001/a8x issue

**Comment by TheRealClarity** [comment]
Oops, my bad. Issue reopened. Thank you.
